src/autowiring/DispatchQueue.cpp

```diff
@@ -23,39 +23,6 @@ DispatchQueue::~DispatchQueue(void) {
   }
 }
 
-void DispatchQueue::Abort(void) {
-  std::lock_guard<std::mutex> lk(m_dispatchLock);
-  m_aborted = true;
-
-  // Do not permit any more lambdas to be pended to our queue:
-  m_dispatchCap = 0;
-
-  // Destroy the whole dispatch queue:
-  while(!m_dispatchQueue.empty()) {
-    delete m_dispatchQueue.front();
-    m_dispatchQueue.pop_front();
-  }
-
-  // Wake up anyone who is still waiting:
-  m_queueUpdated.notify_all();
-}
-
-std::chrono::steady_clock::time_point
-DispatchQueue::SuggestSoonestWakeupTimeUnsafe(std::chrono::steady_clock::time_point latestTime) const {
-  return
-    m_delayedQueue.empty() ?
-
-    // Nothing in the queue, no way to suggest a shorter time
-    latestTime :
-
-    // Return the shorter of the maximum wait time and the time of the queue ready--we don't want to tell the
-    // caller to wait longer than the limit of their interest.
-    std::min(
-      m_delayedQueue.top().GetReadyTime(),
-      latestTime
-    );
-}
-
 void DispatchQueue::PromoteReadyEventsUnsafe(void) {
```

Comment:
Change this return type to `bool` and have the return condition indicate whether any events were promoted.